multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -291,12 +293,17 @@ def validation_step(self, batch, batch_idx):
         student_output, loss = self._shared_step(batch)
         # By default, on_step=False and on_epoch=True
         self.log("val_loss", loss)
+        self._compute_metric_score(
+            metric=self.validation_metric,
+            custom_metric_func=self.custom_metric_func,
+            logits=student_output[self.student_model.prefix][LOGITS],
+            label=batch[self.student_model.label_key],
+        ),
         self.log(
             self.validation_metric_name,
-            self._compute_metric(
-                logits=student_output[self.student_model.prefix][LOGITS],
-                label=batch[self.student_model.label_key],
-            ),
+            self.validation_metric,
+            on_step=False,
+            on_epoch=True,
```

Comment:
I had the concern on whether we should reset the metrics but found this issue: https://github.com/Lightning-AI/lightning/issues/4396 . Thus, the current implementation is correct.